src/components/FaqSection/Faq/Faq.styles.js

```diff
@@ -0,0 +1,116 @@
+import styled from 'styled-components';
```

Comment:
Rename this file to .jsx . JSX files are files which export a component, JS files are just normal javascript files